Services/RotaMe.Services/RolesService.cs

```diff
@@ -24,6 +24,44 @@ public RolesService(RotaMeDbContext context, UserManager<RotaMeUser> userManager
             this.userManager = userManager;
         }
 
+        public async Task<bool> Create(RoleCreateServiceModel roleCreateServiceModel)
+        {
+            var roles = await this.context.Roles.ToListAsync();
+            if (roles.FirstOrDefault(r => r.NormalizedName == roleCreateServiceModel.NarmalizedName) == null)
+            {
+                var result = await context.Roles.AddAsync(new IdentityRole()
+                {
+                    Name = roleCreateServiceModel.Name,
+                    NormalizedName = roleCreateServiceModel.NarmalizedName
+                });
+
+                context.SaveChanges();
+
+                return false ? result == null : true;
+            } else
+            {
+                return false;
+            }
+        }
+
+        public async Task<bool> Delete(string id)
+        {
+            var role = await context.Roles.FindAsync(id);
+
+            var result = context.Roles.Remove(role);
+
+            context.SaveChanges();
+
+            if (result != null)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
```

Comment:
in php we have some helpers for that, looking like -
`return !! result` or `return (bool) result`. I believe you could have something similar.